volume_project/extract/2022/rescu_sqls/rescu_centreline.sql

```diff
@@ -0,0 +1,161 @@
+-- Make a table that links RESCU detectors with the centreline from 2022-07-05
+CREATE TABLE scannon.rescu_cent_20220705 AS (
+
```

Comment:
![image](https://github.com/CityofToronto/bdit_volumes/assets/46324452/7585c19a-45ae-4ce3-8497-19c197b880e5)
`DW0020DWL` can you double check if this is a correct match ?

Reply:
I think I recognize this as a case where there were two sensors with the same description and I picked the one with the higher volume when looking at it last time.

Reply:
`DW0020DWL` is a Lakeshore detector (near Lakeshore and Bay), and it's mapped to the Lakeshore segment that's just west of Bay.